apps/web/src/components/Menu/UserMenu/index.tsx

```diff
@@ -95,7 +95,13 @@ const UserMenu = () => {
 
   if (account) {
     return (
-      <UIKitUserMenu account={sidName || account} avatarSrc={avatarSrc} text={userMenuText} variant={userMenuVariable}>
+      <UIKitUserMenu
+        account={account}
+        sidName={sidName}
```

Comment:
we should remove the `sidName` from uikit
the uikit menu can add a `ellipsis` flag when sidName

Reply:
`<UIKitUserMenu  account={sidName || account} ellipsis={!!sidName}  />`

Reply:
done

Reply:
ellipsis can be default true, or else bridge will show full address

Reply:
done